backstage/community-plugins
# update sidebar layout for argocd plugin

This PR contains following changes

- Redesigned argocd plugin sidebar layout
- Add expand/collapse to resources table
- Show resource metadata in the expanded row
- Refactor and Unit tests.



**Screenshots:**

![Argocd_sidebar](https://github.com/user-attachments/assets/afe02f8b-489b-4abe-bd67-1fd784a4a7a9)


**How to test:**

1. run `yarn start:dev` from `workspaces/redhat-argocd`
2. click on `quarkus-app-prod` application to see the sidebar changes.

#### :heavy_check_mark: Checklist
